package.json

```diff
@@ -8,7 +8,7 @@
     "coverage": "solidity-coverage",
     "deploy-rinkeby": "truffle migrate --network rinkeby",
     "deploy-mainnet": "truffle migrate --network mainnet",
-    "test": "npm run lint ./ && truffle test",
+    "test": "truffle test",
```

Comment:
we're going to have to add the linter back and fix all the linting errors in our PR